roles/ceph-common/tasks/misc/system_tuning.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  failed_when: false
+
+- name: disable transparent hugepage
+  command: "echo never > /sys/kernel/mm/transparent_hugepage/enabled"
+  changed_when: false
+  failed_when: false
+  when: disable_transparent_hugepage
+
+- name: disable swap
+  command: swapoff -a
+  changed_when: false
+  failed_when: false
+  when: disable_swap
+
+- name: apply operating system tuning
+  sysctl: name={{ item.name }}
+          value={{ item.value }}
+          state=present
+          sysctl_file=/etc/sysctl.conf
+          ignoreerrors=yes
+  with_items: os_tuning_params
```

Comment:
Ah, this could be done as a handler, referencing the custom file in /etc/sysctl.d/.